Experience.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <a id = "myName" href="index.html">David Gonzalez</a>
+    </header>
+    <main class="main">
```

Comment:
be careful with the size of the main containers (in all HTML files) interfering with how the side nav looks. on the bio page, for example, the font size was larger than other pages and pushed over the nav, making it look more squished.